gunrock/applications/sssp/sssp_implementation.hxx

```diff
@@ -153,19 +178,19 @@ struct sssp_enactor_t : enactor_t<algorithm_problem_t> {
     operators::advance::execute<operators::advance_type_t::vertex_to_vertex,
                                 operators::advance_direction_t::forward,
                                 operators::load_balance_t::merge_path>(
-        G, enactor_type::get_enactor(), shortest_path, context);
+        G, enactor_t::get_enactor(), shortest_path, context);
 
     // Execute filter operator on the provided lambda
     operators::filter::execute<operators::filter_type_t::predicated>(
-        G, enactor_type::get_enactor(), remove_completed_paths);
+        G, enactor_t::get_enactor(), remove_completed_paths);
   }
 
-  sssp_enactor_t(algorithm_problem_t* _problem,
+  sssp_enactor_t(problem_t* _problem,
                  std::shared_ptr<cuda::multi_context_t> _context)
-      : enactor_type(_problem, _context) {}
+      : enactor_t(_problem, _context) {}
 
-  sssp_enactor_t(const sssp_enactor_t& rhs) = delete;
-  sssp_enactor_t& operator=(const sssp_enactor_t& rhs) = delete;
+  sssp_enactor_t(const sssp_enactor_t& rhs) = delete;            // Boilerplate? Can remove?
+  sssp_enactor_t& operator=(const sssp_enactor_t& rhs) = delete; // Boilerplate? Can remove?
```

Comment:
Need to check if restricting ctor/ref in the base class is enough, then we can remove this.

Reply:
ok